.travis.yml

```diff
@@ -65,7 +66,7 @@ jobs:
 
     # Default stage (used with matrix).
     - stage: test
-      python: 3.8-dev
+      python: 3.8
       env: PIP=latest
```

Comment:
It looks like we can completely remove these lines since "3.8" added to the matrix.